### DISCUSSION TITLE
how to use vTaskList with esp32 Arduino and PlatfromIO?

### DISCUSSION THREAD

**Discussion by xBLACKICEx** [espressif/arduino-esp32#9225] [Q&A] [answered]
how to use vTaskList with esp32 arduino and platfromio?

im adding flag on platfromIO build_flags but is still not work, always with error :  undefined reference to `vTaskList'
-DCONFIG_FREERTOS_VTASKLIST_INCLUDE_COREID=1
-DCONFIG_FREERTOS_USE_TRACE_FACILITY=1
-DCONFIG_FREERTOS_USE_STATS_FORMATTING_FUNCTIONS=1

what else do I miss?

**Comment by lucasssvaz** [comment, marked as answer]
You can track this in #7179
